mne/stats/tests/test_cluster_level.py

```diff
@@ -651,11 +651,29 @@ def test_permutation_test_H0(numba_conditional):
         assert_equal(len(h0), 2 ** (7 - (tail == 0)))  # exact test
 
 
-def test_tfce_thresholds(numba_conditional):
-    """Test TFCE thresholds."""
+def _gen_tfce_data(n_dims):
     rng = np.random.RandomState(0)
-    data = rng.randn(7, 10, 1) - 0.5
 
+    if n_dims == 2:
+        data = rng.randn(7, 10)
+    elif n_dims == 3:
+        data = rng.randn(7, 10, 1)
+    else:
+        raise ValueError('Only 2D and 3D arrays supported in the test')
+
+    data -= 0.5
+    return data
+
+
+@pytest.mark.parametrize(
+    'data',
+    (
+        _gen_tfce_data(n_dims=3),
+        _gen_tfce_data(n_dims=2)
+    )
+)
```

Comment:
this looks odd to me.  Since the helper function is *only* being used for this one test, why not generate the data inside the test like before?  Since the cases you're testing are shape (7, 10) and shape (7, 10, 1) you could either parametrize over `n_dims`, or (perhaps even simpler) generate shape (7, 10) data and run the test line, and then add a `np.newaxis` and run it again (this also avoids doing the `with pytest.raises` checks twice).  WDYT?

Reply:
I've reworked the test(s) accordingly, better now?

Reply:
yep, thanks. I pushed commits to fix a changelog typo, and to (nitpickily) streamline this test a tiny bit more.